pkg/republisher/uuid_republisher_test.go

```diff
@@ -49,9 +49,9 @@ var testCollections = Collections{
 }
 
 var testCollectionsSingle = Collections{
-	"methode": {
-		name:                  "methode",
-		defaultOriginSystemID: "methode-web-pub",
+	"universal-content": {
```

Comment:
Shouldn't we add tests for `universal-content` instead of overwriting the `methode` ones?

Reply:
this is exactly what it's doing, it tests with universal-content instead of methode, this struct is used for single case collections, failure or success where the collections do not matter for the outcome.

Above there's a struct with many collections for multicollection tests purposes : testCollections

Reply:
My point is that we should *not* overwrite the `methode` test cases but rather to add new test cases for `universal-content`.
When the an uuid is not found in `universal-content` it should be possible for it to be republished from `methode` collection.

Reply:
Agree, there should be probably a few more cases. 

But in this case we are not overwriting Methode, in a couple of places it really does not matter the name of the single collection but when it fails it does, and since Methode is being decommissioned it should be replaced that logic with universal-content. Refer to : https://github.com/Financial-Times/publish-failure-resolver-go/blob/1aecbe434f72883aa98068de57bf55c84d3d1a7a/pkg/republisher/uuid_republisher.go#L89

Reply:
Maybe there is a slight confusion about what Methode decommissioning means for our pipeline. Supporting republishing of Methode content should be supported long after Methode is decommissioned. This is due to concept concordance updates, and overall being able to update anything metadata related for these pieces of content. So this should be supported.

In this case using `universal-content` as main collection for tests is ok. What I find disturbing is that there are no new test cases for the newly introduced logic. Tests should be extended to cover at least the following few cases:

- **Given** the same `uuid` is present in both `methode` and `universal-content` collections,
  **When** repubslihed with `scopeContent`,
  **Then** it should be republished from `universal-content` collection and not from `methode`.
- **Given** the same `uuid` is present in both `methode` and `universal-content` collections,
  **When** repubslihed with `scopeBoth`,
  **Then** it should be republished from `universal-content` collection and not from `methode`.

- **Given** an `uuid` is present in `methode` collection but not in `universal-content`,
  **When** repubslihed with `scopeContent`,
  **Then** it should be republished from `methode` collection.
- **Given** an `uuid` is present in `methode` collection but not in `universal-content`,
  **When** repubslihed with `scopeBoth`,
  **Then** it should be republished from `methode` collection.

Reply:
the use cases have been covered by tests

Reply:
The tests now cover these use cases.